index.html

```diff
@@ -149,15 +149,17 @@
             [ 310.5 * 202.5, 'Strahovských stadionů'],
             [ 496 * 1000 * 1000, 'rozloh Prahy'],
             [ 696241 * 1000 * 1000, 'rozloh Texasu']
```

Comment:
Chybí čárka, takže to neběhá.

Reply:
Je nutná i za poslední položkou pole (pak by toho ale nefungovalo víc) nebo narážíš na něco jiného?

Reply:
Myslel sem za Texasem - tam je nutná, pač je to seznam. Za posledním elementem listu není povinná (ale je fajn pro čistší diffy).

Reply:
Jo, pravda, já jsem se díval na blbou branch, tam byl Texas poslední, proto mě to zmátlo...